bigtable/client/table_admin.h

```diff
@@ -58,6 +58,10 @@ class TableAdmin {
       : impl_(std::move(client), std::move(instance_id),
               std::move(retry_policy), std::move(backoff_policy)) {}
 
+  TableAdmin(TableAdmin const& table_admin) = default;
+
```

Comment:
Remove the blank line if you can.

Reply:
static_assert is added in file "bigtable/client/internal/table_admin.cc" in last commit